scripts/cc-cli/src/index.ts

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
+
+import { Command } from "commander";
+
+// Subcommands
+import { makeWizardCommand } from "./commands/wizard";
+import { makeNewSeedCommand } from "./commands/newSeed";
+import { makeBalanceCommand } from "./commands/balance";
+import { makeValidateCommand } from "./commands/validate";
+import { makeBondCommand } from "./commands/bond";
+import { makeRotateKeysCommand } from "./commands/rotateKeys";
+import { makeSetKeysCommand } from "./commands/setKeys";
+import { makeReceiveCommand } from "./commands/receive";
+import { makeSendCommand } from "./commands/send";
+import { makeChillCommand } from "./commands/chill";
+
+const program = new Command();
+
+program.version("0.0.1").description("Creditcoin Staking Tool");
```

Comment:
Is version required? If yes, let's try keeping it consistent b/c `index.ts` and `package.json`.